hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -351,6 +381,7 @@ public synchronized void appendNewBlocks(
     if (updateTime) {
       setModificationTime(Time.now());
     }
+    refreshCurrentLocationPartsMap();
```

Comment:
Why is this called in `appendNewBlocks`? `appendNewBlocks` are only called in server side and the `currentlocationsPartsMap` and `currentDataSizePartsMap` are not persisted right?